introduction.md

```diff
@@ -0,0 +1 @@
+Hi! I'm Christine. I like programming because of the many opportunities it offers to learn something new.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.